server/server.go

```diff
@@ -0,0 +1,132 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package server
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/base64"
+	"errors"
+	"github.com/eclipse-xpanse/policy-man/config"
+	"github.com/eclipse-xpanse/policy-man/log"
+	"github.com/gin-contrib/logger"
+	"github.com/gin-gonic/gin"
+	"golang.org/x/sync/errgroup"
+	"net/http"
+	"time"
+)
+
+func RunHTTPServer(ctx context.Context, cfg *config.Conf) error {
+	server := &http.Server{
+		Addr:    cfg.Host + ":" + cfg.Port,
+		Handler: router(cfg),
+	}
+
+	log.Info("HTTP server is running on" + cfg.Host + ":" + cfg.Port)
```

Comment:
we need a space after 'on'
Also the timestamp is having some issue - 
`time="22226/09/09 - 09:09:09" level=info msg="HTTP server is running onlocalhost:8080"
`